README.md

```diff
@@ -0,0 +1,6 @@
+## Product one
```

Comment:
You might want to change the content in the README.md. The skeleton used for the README doc looks good.